documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -805,7 +822,8 @@ When generating the implementation of a mapping method, MapStruct will apply the
 * If source and target attribute have the same type, the value will be simply copied from source to target. If the attribute is a collection (e.g. a `List`) a copy of the collection will be set into the target attribute.
 * If source and target attribute type differ, check whether there is a another mapping method which has the type of the source attribute as parameter type and the type of the target attribute as return type. If such a method exists it will be invoked in the generated mapping implementation.
 * If no such method exists MapStruct will look whether a built-in conversion for the source and target type of the attribute exists. If this is the case, the generated mapping code will apply this conversion.
-* Otherwise an error will be raised at build time, indicating the non-mappable attribute.
+* If no such method was found MapStruct will generate an internal method that will do the mapping between the source and target attribute
```

Comment:
Can we add a sentence that explicitly creating the nested method is encouraged when it needs further configuration. I.e. we encourage not to configure it through the parent method (as enabled now by #1072), the reason being that this puts the configuration of the nested mapping into one place (method) where it can be be re-used from several methods at the upper level, instead of re-configuring the same things on all of those upper methods.